v2/session/serverless/with-aws-lambda/about.mdx

```diff
@@ -4,22 +4,23 @@ title: About
 hide_title: true
```

Comment:
not related to this file, however, the new images in this PR are much larger than the size they will be rendered at. So you can re-upload smaller versions of the images please.

Reply:
done

Reply:
Was this addressed? I don't see image size that are smaller in the PR.

Reply:
Sizes did reduce in [this commit](https://github.com/supertokens/docs/pull/757/commits/0f3cad24cfd6308bddee0b6e63312907729e93d4#diff-8b1f45a8dfe5225a01477362564909552b1f68fcfe40fba03af632f8f4d5f9ec) but I can reduce it further.

Reply:
Unable to reduce it further using ImageOptim software I am using. If you want to reduce it further, do suggest some tool that you use.

Reply:
No i mean that the diff shows that the new images are of the same dimensions as the older images. So not sure whats happening at all. Also, you can reduce image size using Mac's preview software easily (google it)

Reply:
Done